src/main/java/io/github/moulberry/notenoughupdates/overlays/FarmingSkillOverlay.java

```diff
@@ -286,15 +327,66 @@ public void update() {
 			int last = counterQueue.getLast();
 			int first = counterQueue.getFirst();
 
-			cropsPerSecond = (first - last) / 3f;
+			//This is a list of the last X cropsPerSeconds to try and calm down the fluctuation for crops/min (they will be averaged)
+			//Needed due to farming fortune causing inconsistent amounts of crops each block break
+			//Making this while in case somehow it goes over X+1
+			while (cropsOverLastXSeconds.size() > 60) {
+				cropsOverLastXSeconds.remove(0);
+			}
+			if ((first - last) / 3f != 0) {
+				cropsOverLastXSeconds.add((first - last) / 3f);
```

Comment:
changing `(first - last) / 3f` to `(first - last) / 2f` in line 336 and 337 would fix the crops/m counter being off by so much and resolve my PR #640.

Reply:
I *think* I fixed this? I assume this is what you meant.

Reply:
Yep, that should do it.